flink/v1.19/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -349,18 +396,20 @@ private <T> DataStreamSink<T> chainIcebergOperators() {
       // Find out the equality field id list based on the user-provided equality field column names.
       List<Integer> equalityFieldIds = checkAndGetEqualityFieldIds();
 
-      // Convert the requested flink table schema to flink row type.
       RowType flinkRowType = toFlinkRowType(table.schema(), tableSchema);
+      int writerParallelism =
```

Comment:
writer parallelism is also needed by `distributeDataStream` method as the downstream operator parallelism for range partitioner.